developing.adoc

```diff
@@ -0,0 +1,45 @@
+
+
+= Developing CodeReady Containers
+
+[[developing-overview]]
+== Overview
+
+The following sections describe how to build and test {project}.
+
+[[develop-prerequisites]]
+== Prerequisites
+
+- Git
+- Make
+- A recent Go distribution (>=1.11)
+
+[NOTE]
+====
+You should be able to develop the project on Linux, Windows or macOS.
+====
+
+[[set-up-dev-env]]
+== Setting Up the Development Environment
+
+[[cloning-repository]]
+=== Cloning the Repository
+
+Get the sources from GitHub:
+
+----
+$ git clone https://github.com/code-ready/crc.git
+----
+
+Note: Do not keep the source in your $GOPATH as it will cause `make` to fail. Because CRC uses link:https://github.com/golang/go/wiki/Modules[go modules] and it does not expect the source to be in the $GOPATH.
+
+[[manage-dependencies]]
+== Dependency Management
+
+CRC uses link:https://github.com/golang/go/wiki/Modules[go modules] for dependency management.
+
+Refer below links for more information.
+
+. link:https://github.com/golang/go/wiki/Modules[Introduction to go modules]
```

Comment:
The title for this link is good, as it adds more context. However, we need to capitalize Go:

```suggestion
. link:https://github.com/golang/go/wiki/Modules[Introduction to Go modules]
```